about.html

```diff
@@ -0,0 +1,81 @@
+    <link rel="stylesheet" href="about.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="wrapper">
+        <div class="top">
+            <h1>About
+                <br>Myself</h1>
+            <p class="normaltxt"> Hello I'm a software developer! I can help     you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+            
+            <div class="social">
+                <h3>LET'S CONNECT</h3>
+      
+                <ul class="vectorfram">
+                    <li>
+                  <button class="vector-button" type="button">     <img src="./images/github.svg" alt="github"></button>
+                    </li>
+                    <li>
+                     <button class="vector-button" type="button"><img src="./images/linkedin.svg" alt="linked in"></button>
+                    </li>
+                    <li>
+                      <button class="vector-button" type="button"><img src="./images/Vector.svg" alt="vector"></button>
+                    </li>
+                    <li>
+                      <button class="vector-button" type="button"><img src="./images/twitter.svg" alt="twitter"></button>
+                    </li>
+                    <li>
+                      <button class="vector-button" type="button"><img src="./images/medium.svg" alt="m"></button>
+                </ul>
+            </div>
+            <button type="button" class=" button">
+                Get my resume
+            </button>
+    </div>
+  
+    <div class="skill ">
+        <div class="listfram ">
+            <div class="listtag ">
+                <p class="listname">Languages</p>
+            </div>
+            <ul class="skillul ">
+                <li class="skillli ">
+                    <div class="listimg"><img src="./images/FrameJava.svg" alt="javascript"></div>
+                </li>
+                <li class="skillli ">
+                    <div class="listimg"><img src="./images/HTMLFrame10.svg" alt="html"></div>
+                </li>
+                <li class="skillli ">
+                    <div class="listimg"><img src="./images/CSSFrame11(1).svg" alt="CSS"></div>
+                    
+                </li>
+            </ul>
+  
+            <div class="listfram ">
+                <div class="listtag ">
+                    <button type="button" class="icon"><img src="./images/F1.svg.svg" alt="button2"></button>
+                </div>
+                <ul class="skillul">
+                </ul>
+            </div>
+            <div class="listfram ">
+                <div class="listtag ">
+            <button type="button" class="icon"><img src="./images/Frame.svg" alt="button3"></button>
+                </div>
+                <ul class="skillul">
+                </ul>
+            </div>
+  
+        </div>
+    </div>
+  </section>
+  
+
+</body>
+</html>
```

Comment:
- [ ] The about page doesn't have to be in a separate file  with `index.html` as the previous reviewer requested and you are repeating yourself if you can check I suggest that you create the technologies `section` inside `index.html` instead of separating them because when I check your index page I am not able to see tech stack you use, and remember you don't have to repeat information you already have on the index page

Reply:
Hello Lydia. Kindly check well as I effected that changes regarding that section of about page and it is inside my html file. Thanks.